src/lib/callApi.ts

```diff
@@ -0,0 +1,39 @@
+import { dev } from '$app/environment';
+import { get } from 'svelte/store';
+import { xsrf } from './stores';
+
+const callApi = async <
+  ResBody = Record<string, unknown>,
+  ReqBody = Record<string, unknown>
+>(
+  input: string,
+  method: RequestInit['method'] = 'GET',
+  body?: ReqBody,
+  headers?: Request['headers']
+): Promise<{ status: number; body: ResBody }> => {
+  const xsrfToken = get(xsrf);
+
+  const res = await fetch('/api' + input, {
+    headers: {
+      ...headers,
+      'Content-Type': body ? 'application/json' : undefined,
+      'X-XSRF-TOKEN': xsrfToken,
+    },
+    credentials: dev ? 'include' : 'same-origin',
```

Comment:
How does this work? :thinking:

Reply:
aiai nice catch. It includes headers when using different origins, removing it since the proxy fixes the (actual) issue:3